ansible/setup.yml

```diff
@@ -34,9 +34,9 @@
         - project-user
 
     # Automatic cleanup is not supported. You must remove old keys manually.
-    - name: "Setup root SSH authorized keys"
+    - name: "Setup master SSH authorized keys"
       authorized_key:
-        user: "{{ ansible_user }}"
+        user: master
```

Comment:
I understand CI runs this script every day on existing environments. What is impact?

Reply:
Should have no impact for CI runs. This is fix for packer builds where the playbook is run with ‘ubuntu’ user for AWS builds